cc_licenses/templates/includes/use_of_licenses.html

```diff
@@ -21,7 +21,7 @@ <h4 class="level is-vcentered b-header">{% trans "Considerations for licensors"
       and conditions of the license they choose before applying it. Licensors should also secure all rights necessary before applying
       our licenses so that the public can reuse the material as expected. Licensors should clearly mark any material not subject to the license.
       This includes other CC-licensed material, or material used under an exception or limitation to copyright.
-      <a href="#">More considerations for licensors.</a>
+      <a href="https://wiki.creativecommons.org/wiki/Considerations_for_licensors_and_licensees#Considerations_for_licensors">More considerations for licensors.</a>
```

Comment:
It looks as if a couple of unrelated changes in this file snuck into the PR.

Reply:
yes, do you want me to remove this link? I put it here from my chat with vocab team.

Reply:
If it was intentional, not a big deal.

Reply:
got it.